pkg/manifest/ostree_deployment.go

```diff
@@ -197,6 +193,11 @@ func (p *OSTreeDeployment) serialize() osbuild.Pipeline {
 		)
 	}
 
+	pipeline.AddStage(osbuild.NewOSTreePullStage(
+		&osbuild.OSTreePullStageOptions{Repo: repoPath, Remote: p.Remote.Name},
+		osbuild.NewOstreePullStageInputs("org.osbuild.source", commit.Checksum, commit.Ref),
+	))
+
```

Comment:
We don't really need a pull stage at all anymore since https://github.com/osbuild/osbuild/pull/1393 - just specify the commit as an input to the deploy stage.

Reply:
I want to keep the existing images unchanged while adding support for the new stuff to avoid any side effects or confusion.  I intend to drop the pull stage in a follow-up, but for now I'd like the existing images to change as little as possible and so I'm limiting their changes to just a stage reordering.

Reply:
ok